pom.xml

```diff
@@ -171,7 +171,7 @@
     <xstream.version>1.4.20</xstream.version>
     <jackrabbit.version>2.21.19</jackrabbit.version>
     <oak-jackrabbit.version>1.48.0</oak-jackrabbit.version>
-    <commons-compress.version>1.24.0</commons-compress.version>
+    <commons-compress.version>1.26.1</commons-compress.version>
```

Comment:
1.26.2 is the latest. Consider using this instead.

Reply:
Additionally, version 1.26.2 is the correct version to use with commons-io 2.16.1.

<img width="1360" alt="Screenshot 2024-06-17 102506" src="https://github.com/pentaho/maven-parent-poms/assets/367863/d60843bf-a1a0-40e0-9052-250596a9627d">